### PR TITLE
WebXRManager: Set up initial input sources, if any.

### LINKED ISSUE #18638: Don't assume that an inputsourceschange event will be fired at the start of the session
##### Description of the problem

Currently, the three.js code setting up XR controllers only does this setup when an `inputsourceschange` event is fired. However, it is possible for a session to start with controllers already attached. My understanding is that Chrome will always fire `inputsourceschange` events at the beginning of the session, but I don't think this is necessary according to the spec. Servo currently just starts off with the correct input sources, only spawning events when things change.

We should instead initialize the controller map when the session is created.

https://github.com/mrdoob/three.js/blob/fb29d21e65c2cb8a0bf8b797c0deefa5fe8c8cde/src/renderers/webxr/WebXRManager.js#L233

##### Three.js version

- [x] Dev
- [ ] r113
- [ ] ...

##### Browser

- [x] All of them
- [ ] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [x] All of them
- [ ] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

##### Hardware Requirements (graphics card, VR Device, ...)

 - VR Device

### PR DESCRIPTION
**Pull request by Manishearth**
Fixes #18638

I'm not sure if I should also run `build` and check that in as well.

One potential cleanup here is to factor out the `if (controller) ...` bits into a new function.

r? @mrdoob

### CODE REVIEW

**Comment by mrdoob** [comment]
> I'm not sure if I should also run `build` and check that in as well.

Better not to include the builds. Otherwise the PR will have conflicts in no time.

**Comment by Manishearth** [comment]
Note: we shouldn't land this till immersive-web/webxr#961 resolves

**Comment by Mugen87** [comment]
>Better not to include the builds

The build files are still part of this PR.

File: src/renderers/webxr/WebXRManager.js [hunk 1]
```diff
@@ -221,7 +221,27 @@ function WebXRManager( renderer, gl ) {
...
+					}
+
+					if ( controller.grip ) {
+
+						controller.grip.dispatchEvent( { type: 'connected', data: inputSources[ i ] } );
+
+					}
+
+				}
+			}
```
Comment (Mugen87) [inline src/renderers/webxr/WebXRManager.js]: Please wrap this `}`  in blank lines to make Travis happy.

**Comment by Mugen87** [comment]
Closing, see https://github.com/mrdoob/three.js/issues/18638#issuecomment-623970620.

**Pull request closed without merging**